### PR TITLE
Use MDN data to generate :autofill support data

### PR DESCRIPTION
**Pull request by lukewarlow**
This uses MDN's browser compat data npm package to get the support data for the `:autofill` Pseudo class.

This is required because caniuse won't add entries that are duplicated with MDN. And autoprefixer requires the data to generate a prefix see https://github.com/postcss/autoprefixer/issues/626.

Not exactly sure on the exact output format required but so far I've got this JSON being generated.

image

### CODE REVIEW

**Comment by lukewarlow** [comment]
Are there utils I should look at which are used for generating the tiny output data? If so I'm happy to get this so it's generating JSON in the right format and putting it thought that util to minify it.

File: check.js [hunk 1]
```diff
@@ -27,3 +28,50 @@ get('https://registry.npmjs.org/caniuse-db', res => {
...
+  return browser
+}
+
+let mappedData = {}
+Object.keys(autofillSupportData).forEach(browser => {
+  let browserDataRaw = autofillSupportData[browser]
+  let browserData
+
+  if (Array.isArray(browserDataRaw)) {
+    if (browserDataRaw[0].version_added === 'preview') {
```
Comment (lukewarlow) [inline check.js:60]: This is to account for Safari TP data which afaik this project doesn't handle.

**Comment by lukewarlow** [comment]
Managed to get this as the output format so far.

```json
{
   "title":":autofill CSS pseudo-class",
   "spec":"https://html.spec.whatwg.org/multipage/semantics-other.html#selector-autofill",
   "stats":{
      "chrome":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "12":"n",
         "13":"n",
         "14":"n",
         "15":"n",
         "16":"n",
         "17":"n",
         "18":"n",
         "19":"n",
         "20":"n",
         "21":"n",
         "22":"n",
         "23":"n",
         "24":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "31":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "69":"n",
         "70":"n",
         "71":"n",
         "72":"n",
         "73":"n",
         "74":"n",
         "75":"n",
         "76":"n",
         "77":"n",
         "78":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"n",
         "87":"n",
         "88":"n",
         "89":"n",
         "90":"n",
         "91":"n",
         "92":"n",
         "93":"n",
         "94":"n",
         "95":"n",
         "96":"n"
      },
      "and_chr":{
         "18":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "31":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "69":"n",
         "70":"n",
         "71":"n",
         "72":"n",
         "73":"n",
         "74":"n",
         "75":"n",
         "76":"n",
         "77":"n",
         "78":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"n",
         "87":"n",
         "88":"n",
         "89":"n",
         "90":"n",
         "91":"n",
         "92":"n",
         "93":"n",
         "94":"n",
         "95":"n",
         "96":"n"
      },
      "edge":{
         "12":"n",
         "13":"n",
         "14":"n",
         "15":"n",
         "16":"n",
         "17":"n",
         "18":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"n",
         "87":"n",
         "88":"n",
         "89":"n",
         "90":"n",
         "91":"n",
         "92":"n",
         "93":"n",
         "94":"n",
         "95":"n",
         "96":"n"
      },
      "firefox":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "12":"n",
         "13":"n",
         "14":"n",
         "15":"n",
         "16":"n",
         "17":"n",
         "18":"n",
         "19":"n",
         "20":"n",
         "21":"n",
         "22":"n",
         "23":"n",
         "24":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "31":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "69":"n",
         "70":"n",
         "71":"n",
         "72":"n",
         "73":"n",
         "74":"n",
         "75":"n",
         "76":"n",
         "77":"n",
         "78":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "82":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"y",
         "87":"y",
         "88":"y",
         "89":"y",
         "90":"y",
         "91":"y",
         "92":"y",
         "93":"y",
         "94":"y",
         "95":"y",
         "96":"y",
         "97":"y",
         "98":"y",
         "1.5":"n",
         "3.5":"n",
         "3.6":"n"
      },
      "and_ff":{
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "14":"n",
         "15":"n",
         "16":"n",
         "17":"n",
         "18":"n",
         "19":"n",
         "20":"n",
         "21":"n",
         "22":"n",
         "23":"n",
         "24":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "31":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "82":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"y",
         "87":"y",
         "88":"y",
         "89":"y",
         "90":"y",
         "91":"y",
         "92":"y",
         "93":"y",
         "94":"y",
         "95":"y",
         "96":"y",
         "97":"y",
         "98":"y"
      },
      "ie":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "5.5":"n"
      },
      "opera":{
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "12":"n",
         "15":"n",
         "16":"n",
         "17":"n",
         "18":"n",
         "19":"n",
         "20":"n",
         "21":"n",
         "22":"n",
         "23":"n",
         "24":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "31":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "60":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "69":"n",
         "70":"n",
         "71":"n",
         "72":"n",
         "73":"n",
         "74":"n",
         "75":"n",
         "76":"n",
         "77":"n",
         "78":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "3.5":"n",
         "3.6":"n",
         "5.1":"n",
         "7.1":"n",
         "7.2":"n",
         "7.5":"n",
         "8.5":"n",
         "9.1":"n",
         "9.2":"n",
         "9.5":"n",
         "9.6":"n",
         "10.1":"n",
         "10.5":"n",
         "10.6":"n",
         "11.1":"n",
         "11.5":"n",
         "11.6":"n",
         "12.1":"n"
      },
      "op_mob":{
         "11":"n",
         "12":"n",
         "14":"n",
         "15":"n",
         "16":"n",
         "18":"n",
         "19":"n",
         "20":"n",
         "21":"n",
         "22":"n",
         "24":"n",
         "25":"n",
         "26":"n",
         "27":"n",
         "28":"n",
         "29":"n",
         "30":"n",
         "32":"n",
         "33":"n",
         "34":"n",
         "35":"n",
         "36":"n",
         "37":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "10.1":"n",
         "11.1":"n",
         "11.5":"n",
         "12.1":"n"
      },
      "safari":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "12":"n",
         "13":"n",
         "14":"n",
         "15":"n",
         "1.1":"n",
         "1.2":"n",
         "1.3":"n",
         "3.1":"n",
         "5.1":"n",
         "6.1":"n",
         "9.1":"n",
         "10.1":"n",
         "11.1":"n",
         "12.1":"n",
         "13.1":"n",
         "14.1":"n",
         "15.1":"n"
      },
      "ios_saf":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "5":"n",
         "6":"n",
         "7":"n",
         "8":"n",
         "9":"n",
         "10":"n",
         "11":"n",
         "12":"n",
         "13":"n",
         "14":"n",
         "15":"n",
         "3.2":"n",
         "4.2":"n",
         "6.1":"n",
         "9.3":"n",
         "10.3":"n",
         "11.3":"n",
         "12.2":"n",
         "13.4":"n",
         "14.5":"n"
      },
      "samsung":{
         "1.0":"n",
         "1.5":"n",
         "1.6":"n",
         "2.0":"n",
         "2.1":"n",
         "3.0":"n",
         "3.2":"n",
         "4.0":"n",
         "4.2":"n",
         "5.0":"n",
         "5.2":"n",
         "5.4":"n",
         "6.0":"n",
         "6.2":"n",
         "6.4":"n",
         "7.0":"n",
         "7.2":"n",
         "7.4":"n",
         "8.0":"n",
         "8.2":"n",
         "9.0":"n",
         "9.2":"n",
         "9.4":"n",
         "10.0":"n",
         "10.2":"n",
         "11.0":"n",
         "11.2":"n",
         "12.0":"n",
         "12.1":"n",
         "13.0":"n",
         "13.2":"n",
         "14.0":"n",
         "14.2":"n",
         "15.0":"n",
         "16.0":"n"
      },
      "android":{
         "1":"n",
         "2":"n",
         "3":"n",
         "4":"n",
         "37":"n",
         "38":"n",
         "39":"n",
         "40":"n",
         "41":"n",
         "42":"n",
         "43":"n",
         "44":"n",
         "45":"n",
         "46":"n",
         "47":"n",
         "48":"n",
         "49":"n",
         "50":"n",
         "51":"n",
         "52":"n",
         "53":"n",
         "54":"n",
         "55":"n",
         "56":"n",
         "57":"n",
         "58":"n",
         "59":"n",
         "60":"n",
         "61":"n",
         "62":"n",
         "63":"n",
         "64":"n",
         "65":"n",
         "66":"n",
         "67":"n",
         "68":"n",
         "69":"n",
         "70":"n",
         "71":"n",
         "72":"n",
         "73":"n",
         "74":"n",
         "75":"n",
         "76":"n",
         "77":"n",
         "78":"n",
         "79":"n",
         "80":"n",
         "81":"n",
         "83":"n",
         "84":"n",
         "85":"n",
         "86":"n",
         "87":"n",
         "88":"n",
         "89":"n",
         "90":"n",
         "91":"n",
         "92":"n",
         "93":"n",
         "94":"n",
         "95":"n",
         "96":"n",
         "1.5":"n",
         "2.2":"n",
         "4.4":"n",
         "4.4.3":"n"
      }
   }
}
```

**Comment by ai** [comment]
Next steps:

1. Move code from `check.js` to `copy-mdn.js`
2. Call `copy-mdn.js` before `Pack Can I Use data` in `.github/workflows/check.yml`
3. Try to update `@mdn/browser-compat-data` in Yarn lockfile during `.github/workflows/check.yml` before calling `copy-mdn.js`

File: check-mdn-bcd.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  let data = ''
+  res.on('data', chunk => {
+    data += chunk
+  })
+  res.on('end', () => {
+    let body = JSON.parse(data)
+    let lastVersion = body['dist-tags'].latest
+    if (pkg.devDependencies['@mdn/browser-compat-data'] !== lastVersion) {
```
Comment (lukewarlow) [inline check-mdn-bcd.js:19]: Seeing as MDN uses semver potentially should only update minor and patch versions automatically?

File: check-mdn-bcd.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      pkg.devDependencies['@mdn/browser-compat-data'] = lastVersion
+      writeFile('./package.json', `${JSON.stringify(pkg, null, 2)}\n`, () => {
```
Comment (lukewarlow) [inline check-mdn-bcd.js:21]: Only updates package.json currently presumably the workflow will update the lock when it runs install?

File: .github/workflows/check.yml [hunk 4]
```diff
@@ -19,14 +19,19 @@ jobs:
...
+        id: mdn
+        run: node ./check.js
       - name: Setup git config
         if: steps.caniuse.outputs.newVersion
         run: |
           git config user.name "Auto-Updater"
           git config user.email "<andrey+caniuse@sitnik.ru>"
       - name: Install dependencies
-        if: steps.caniuse.outputs.newVersion
+        if: steps.caniuse.outputs.newVersion || steps.mdn.outputs.newVersion
```
[the remark this answers is not in the document]
Comment (ai) [reply .github/workflows/check.yml:31]: Our version is connected to `caniuse-db`. So let's use only `steps.caniuse.outputs.newVersion`

File: check-mdn-bcd.js (hunk 2 above)
Comment (ai) [reply check-mdn-bcd.js:19]: Yeap. And throw an error on major update.

I will do it manually.

Comment (ai) [reply check-mdn-bcd.js:19]: Thanks for noticing edge cases

File: check-mdn-bcd.js (hunk 3 above)
Comment (ai) [reply check-mdn-bcd.js:21]: It will be good to call `yarn install` to test new data with fresh changes

Comment (lukewarlow) [reply check-mdn-bcd.js:31]: My change to check.yml was intended to trigger yarn install when either caniuse or mdn has a new version. Is that not what you intended?

File: .github/workflows/check.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (lukewarlow) [reply .github/workflows/check.yml:31]: My change to check.yml was intended to trigger yarn install when either caniuse or mdn has a new version. Is that not what you intended?

File: .github/workflows/check.yml [hunk 5]
```diff
@@ -19,14 +19,19 @@ jobs:
       - name: Check caniuse-db version
         id: caniuse
         run: node ./check.js
+      - name: Check @mdn/browser-compat-data version
+        id: mdn
+        run: node ./check.js
```
Comment (ai) [inline .github/workflows/check.yml:24]: Should we rename it to `check-mdn-bcd.js`?

File: check-mdn-bcd.js (hunk 3 above)
Comment (ai) [reply check-mdn-bcd.js:31]: You were right. We are calling `yarn install` after `check-mdn-bcd`.

File: copy-mdn.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+       * The BCD version isn't "preview", used for preview browsers such as Safari TP.
+       * The current version is greater than or equal to the BCD version.
+       * There's no prefix information in the BCD entry.
+       * There's no flag information in the BCD entry.
+       * The implementation isn't marked as partial.
+       */
+      let supported = browserData.version_added
+      supported &&= browserData.version_added !== 'preview'
+      supported &&= parseFloat(version) >= parseFloat(browserData.version_added)
+      supported &&= !browserData.prefix
```
Comment (ai) [inline copy-mdn.js:75]: Do we need to add ` x` postfix to `result.stats[bcdBrowserToCanIUseBrowser(browser)][version]` when we need prefix?

Comment (ai) [reply copy-mdn.js:75]: You can find these one-letter codes description here https://github.com/Fyrd/caniuse/blob/main/CONTRIBUTING.md

File: .github/workflows/check.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (ai) [reply .github/workflows/check.yml:31]: Yeap. We can even avoid calling `./check-mdn-bcd.js` on `steps.caniuse.outputs.newVersion === false` to make CI faster on no `caniuse-db` release.

File: copy-mdn.js (hunk 6 above)
Comment (ai) [reply copy-mdn.js:75]: Yeap, for Chrome 96 we need `"y x"` value.

Autoprefixer inserts prefix only if string contain `/ x($|\s)/`

File: .github/workflows/check.yml [hunk 7]
```diff
@@ -27,6 +31,8 @@ jobs:
       - name: Install dependencies
         if: steps.caniuse.outputs.newVersion
         run: yarn install --ignore-scripts
+      - name: Copy MDN data
```
Comment (ai) [inline .github/workflows/check.yml:34]:
```suggestion
      - name: Copy MDN data
      - if: steps.caniuse.outputs.newVersion
```

File: copy-mdn.js (hunk 6 above)
Comment (lukewarlow) [reply copy-mdn.js]: So I've changed the code to make it so if the support statement has a version_added entry and a prefix entry it will add the x otherwise it won't. 

So in the above example it should generate "n x" for all chrome versions up to and including Chrome 95, but for 96 onwards it will generate "y". Which I think is correct behaviour? If we need to handle situations where both prefix and unprefix is supported that will require changing the processing code?

Comment (ai) [reply copy-mdn.js]: Can I Use uses `y x` if browser support a feature but need a prefix.

Comment (lukewarlow) [reply copy-mdn.js]: Will double check the code, on second thoughts I think I may have been wrong above and it's behaving as required after my change

Comment (ai) [reply copy-mdn.js]: `"a x"` is for partial support with prefix

Comment (lukewarlow) [reply copy-mdn.js]: I've updated the code to handle these more complicated scenarios as well as some edge cases that can occur in MDN data such as "range" values (e.g. `≤18`) and `true` for version_added, and the version_removed column for if a feature is no longer supported.

Comment (ai) [reply copy-mdn.js]: Looks good. Can you show the current output after decompressing?

File: data/features/css-autofill.js [hunk 8]
```diff
@@ -0,0 +1 @@
...l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T mB U V W nB oB"},M:{"1":"X Y Z a b c R d H fB pB qB undefined","2":"0 1 2 3 4 5 6 7 8 9 I e J E F G A L D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB P Q T mB U V W"},A:{"2":"undefined lB cB I e J E F G A B kB"},F:{"33":"0 1 2 3 4 5 6 7 8 9 D M N O f g h i j k l m n o p q r s t u v w x y z AB BB CB DB EB FB GB HB IB JB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T","34":"lB cB I e J E F G A B C nB oB sB uB undefined vB ZC hB 1B 2B aB iB 3B bB"},K:{"33":"1 2 3 4 5 6 7 8 9 L D M O f g h i k l m n o p q s t u v w x AB BB CB DB EB FB GB HB IB dB JB eB KB LB S","34":"B C hB aB iB bB"},E:{"33":"undefined cB I e J E F G A B C K L D rB sB tB vB hB aB bB wB xB","34":"lB"},G:{"33":"undefined lB cB I e J E F G A B C K L D gB tB AC CC"},P:{"33":"undefined OC gB YC ZC bC bB cC IC dC eC"},I:{"33":"0 1 2 3 4 5 6 7 8 9 undefined lB cB I x y z AB BB CB DB EB FB GB HB IB dB JB eB KB LB S MB NB OB PB QB RB SB TB UB VB WB XB YB ZB P Q T U V W X Y Z a b c R d H fB pB PC SC"}},B:NaN,C:":autofill CSS pseudo-class"};
```
Comment (ai) [inline data/features/css-autofill.js:1]: Strange that we have `undefined` in the packed array and `NaN`.

Seems like we are doing something wrong.

I am not very familiar with `packager` code, but you can try to put `console.log` or `debugger` to https://github.com/browserslist/caniuse-lite/blob/main/src/packer/feature.js#L80

and to https://github.com/browserslist/caniuse-lite/blob/main/src/packer/feature.js#L99

**Comment by lukewarlow** [comment]
https://gist.github.com/lukewarlow/65d234b378e57b9153421b6ba65da188 - this is the data converted, packed, and then unpacked.

File: data/features/css-autofill.js (hunk 8 above)
Comment (ai) [reply data/features/css-autofill.js:1]: One of the error could created by this line https://gist.github.com/lukewarlow/65d234b378e57b9153421b6ba65da188#file-output-js-L669

Comment (lukewarlow) [reply data/features/css-autofill.js:1]: I believe the NaN is caused by the lack of a "status" value in the caniuse formatted data. It's rather hard to determine the status of the spec for an MDN entry because it only has a `spec_url` and a `standards_track` boolean value. I wouldn't be able to map those to statuses without making some assumptions probably. I'd be curious to know what caniuse treats WICG URLs as too.

Comment (ai) [reply data/features/css-autofill.js:1]: We can use `status: "other"`

Comment (lukewarlow) [reply data/features/css-autofill.js:1]: So the undefined I think is coming from a mismatch between browser versions in MDN and browser versions in caniuse.

Comment (lukewarlow) [reply data/features/css-autofill.js]: The mdn data should now only generate an entry for a version where the version matches a caniuse version. And should have a status of other

**Comment by ai** [comment]
Released in `caniuse-lite@1.0.30001270`
